robosat/tools/templates/leaflet.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
We already have a map client with

https://github.com/mapbox/robosat/blob/master/robosat/tools/templates/map.html

using Mapbox GL JS where we can attach raster layer endpoints to.

What is the reason for pulling in a Leaflet based solution?